Pully Penelope/Assets/Scripts/PlayerMovement.cs

```diff
@@ -131,7 +134,7 @@ private void MovePlayer()
             }
             else if (handleObject.CompareTag("Enemy"))
             {
-                handleObject.GetComponent<Rigidbody2D>().bodyType = RigidbodyType2D.Kinematic;
+                handleObject.GetComponent<Rigidbody2D>().bodyType = RigidbodyType2D.Dynamic;
```

Comment:
using GetComponent in the Update method is something you should try to avoid!

Reply:
Thanks, I will look into changing this.